spec/unit/lib/temporal/grpc_client_spec.rb

```diff
@@ -4,20 +4,28 @@
   let(:namespace) { 'test-namespace' }
   let(:workflow_id) { SecureRandom.uuid }
   let(:run_id) { SecureRandom.uuid }
-  let(:now) { Time.now}
+  let(:now) { Time.now.utc }
+  let(:already_started_error) do
```

Comment:
Will this work with the older server version? Do we have a spec to prove it?

Reply:
What specific older version should I test against? Looking at the [proto definitions](https://github.com/temporalio/api/blame/master/temporal/api/errordetails/v1/message.proto#L45) this error has been around for at least a couple years

Reply:
@cvanderschuere I meant the one with the original unchanged message that the old code was written for